packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx

```diff
@@ -4,28 +4,44 @@ import {
 } from '@coveo/atomic-storybook-utils/commerce/commerce-interface-wrapper';
 import {parameters} from '@coveo/atomic-storybook-utils/common/common-meta-parameters';
 import {renderComponent} from '@coveo/atomic-storybook-utils/common/render-component';
-import {CommerceEngineConfiguration} from '@coveo/headless/dist/definitions/commerce.index';
 import type {Meta, StoryObj as Story} from '@storybook/web-components';
 import {html} from 'lit-html/static.js';
 
 const {decorator, play} = wrapInCommerceInterface({skipFirstSearch: false});
 
-const noResultsEngineConfig: Partial<CommerceEngineConfiguration> = {
-  preprocessRequest: (r) => {
-    const parsed = JSON.parse(r.body as string);
-    // eslint-disable-next-line @cspell/spellchecker
-    parsed.query = 'xqyzpqwlnftguscaqmzpbyuagloxhf';
-    r.body = JSON.stringify(parsed);
-    return r;
-  },
-};
-
-const {play: playNoresults} = wrapInCommerceInterface({
+const {play: playNoProducts} = wrapInCommerceInterface({
   skipFirstSearch: false,
-  engineConfig: noResultsEngineConfig,
+  engineConfig: {
+    preprocessRequest: (r) => {
+      const parsed = JSON.parse(r.body as string);
+      parsed.query = 'show me no products';
+      r.body = JSON.stringify(parsed);
+      return r;
+    },
+  },
 });
 
 const meta: Meta = {
+  argTypes: {
+    'attributes-display': {
+      options: ['grid', 'list', 'table'],
+      control: {type: 'radio'},
+    },
+    'attributes-density': {
+      options: ['compact', 'comfortable', 'normal'],
+      control: {type: 'radio'},
+    },
+    'attributes-image-size': {
+      options: ['small', 'large', 'icon', 'none'],
```

Comment:
At the moment, the atomic-commerce-product-children component doesn't honor imageSize: none. I'll create a Jira.

Reply:
See https://coveord.atlassian.net/browse/KIT-3608